### ISSUE TITLE
5 Nights at Mom’s achievement is broken

### ISSUE THREAD

**Issue by gitBoy2000** [UnderscoreKilburn/repentance-issues#558]
I can’t, for whatever reason unlock the 5 nights at mom’s achievement. I did a 3 win streak with Isaac, Azazel and Cain twice, but still didn’t unlock it. Others have had the same issue.

**Comment by eyhung** [comment]
I just unlocked it with a 3-win streak on normal, at the same time as Ace of Clubs: Isaac, Lazarus, Cain.

**Comment by gitBoy2000** [comment]
are you playing on pc or console? I now believe it may be a console issue

**Comment by eyhung** [comment]
pc

**Comment by gitBoy2000** [comment]
yeah this is definitely a bug but my I got the achievement with ease on my
two pc saves so it’s probably something to do with console. Anyway thanks

**Comment by Rchardon** [comment]
Fixed in Rep+
